configuration/compatibility-testing/test-rp-msa-5.0.2.yml

```diff
@@ -0,0 +1,91 @@
+# Config for MSA 5.0.2-5.0.2 deployed to staging on paas
```

Comment:
The version range mentioned here is confusing. What are we doing?

Reply:
Yeah, that could/should have just said 5.0.2.

The reason it's confusing is because of the way we build MAS artifacts and store them. When we had Jenkins we'd name the artifacts something like `msa-4.1.0-982` where the `4.1.0` is the version we specified in `settings.gradle`. This is what be change when we want to create a release for RP's. The `982` was the Jenkins build number and would be incremented with every build.
We're now building MSA on Concourse and are using a semver resource to keep track of the builds - we increment the patch version of the semver every build. So that's being used as the second part of the artifact name instead of the Jenkins build number. It's also set up so that if the `settings.gradle` version is incremented (normally major or minor) the Concourse semver version will "catch up".
So the `5.0.2-5.0.2` is the first build of version `5.0.2` of the MSA. The next time a PR is merged it'll become `5.0.2-5.0.3`.
The only place this is actually used is in the S3 bucket storing the build zips and is never really seen. The `5.0.2` release is tagged in the repo, and that's all the RP's ever see.